<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uddac2016::PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="negativeSlackColorRangeBeginLabel">
       <property name="text">
        <string>Negative Slack Color Range Begin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLineEdit" name="negativeSlackColorRangeBeginLineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="negativeSlackColorRangeBeginPickColorButton">
         <property name="text">
          <string>Pick Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="negativeSlackColorRangeEndLabel">
       <property name="text">
        <string>Negative Slack Color Range End</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLineEdit" name="negativeSlackColorRangeEndLineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="negativeSlackColorRangeEndPickColorButton">
         <property name="text">
          <string>Pick Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="zeroSlackColorLabel">
       <property name="text">
        <string>Zero Slack Color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLineEdit" name="zeroSlackColorLineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="zeroSlackColorPickColorButton">
         <property name="text">
          <string>Pick Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="positiveSlackColorLabel">
       <property name="text">
        <string>Positive Slack Color</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLineEdit" name="positiveSlackColorLineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="positiveSlackColorPickColorButton">
         <property name="text">
          <string>Pick Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
